src/main/java/org/spongepowered/common/command/SpongeCommandManager.java

```diff
@@ -141,6 +141,9 @@ public SpongeCommandManager(Logger logger, Disambiguator disambiguator) {
             // <namespace>:<alias> for all commands
             List<String> aliasesWithPrefix = new ArrayList<>(aliases.size() * 3);
             for (String alias : aliases) {
+                if (alias.contains(" ")) {
+                    throw new IllegalArgumentException("Plugin " + container.getId() + " attempted to register an invalid alias ('" + alias + "'). Aliases may not contain spaces.");
```

Comment:
`"Plugin '" + container.getId() + "' attempted to register an alias (''" + alias + "'') which contained a space."`